apps/meteor/client/views/room/Header/Header.tsx

```diff
@@ -46,7 +46,15 @@ const Header = ({ room }: HeaderProps<IRoom>): ReactElement | null => {
 		return <VoipRoomHeader slots={slots} room={room} />;
 	}
 
-	return <RoomHeader slots={slots} room={room} topic={room.topic} />;
+	if (shouldDisplayE2EESetup) {
+		return <RoomHeaderE2EESetup room={room} topic={room.topic} slots={slots} />;
+	}
+
+	if (room.t === 'd' && (room.uids?.length ?? 0) < 3) {
```

Comment:
There is a helper for this check `isDirectMessageRoom`: https://github.com/RocketChat/Rocket.Chat/blob/develop/packages/core-typings/src/IRoom.ts#L139

Also, what uids being greater than 3 mean? Would it be better to have this in a named variable maybe?